pepper-apis/src/main/java/org/broadinstitute/ddp/model/study/Participant.java

```diff
@@ -24,6 +25,7 @@
     // todo: better models for status and medical providers
     private EnrollmentStatusDto status;
     private List<MedicalProviderDto> providers;
+    private AgeOfMajorityRule ageOfMajorityRule;
```

Comment:
And if we're really putting stuff about age-up into this `Participant` model, will it make more sense to have the actual date-of-majority `LocalDate` instead of the `AgeOfMajorityRule`?